presto-main/src/main/java/io/prestosql/sql/planner/optimizations/SymbolMapper.java

```diff
@@ -58,7 +61,36 @@
 
     public SymbolMapper(Map<Symbol, Symbol> mapping)
     {
-        this.mapping = ImmutableMap.copyOf(requireNonNull(mapping, "mapping is null"));
```

Comment:
Is this going to impact the planner performance? How often do we create `SymbolMapper` from plain Map in planner?

Reply:
Do you think it is worth to check this with `io.prestosql.sql.planner.BenchmarkPlanner`?